### PR TITLE
introduce release script

### PR DESCRIPTION
**Pull request by jimirocks**

### CODE REVIEW

**Comment by matejhasul** [comment]
LGTM

File: release.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+if [ -z "${gpg_pass}" ]
+then
+   echo "No gpg password given!"
+   usage
+   exit 1
+fi
+
+mvn --batch-mode release:prepare release:perform -DreleaseVersion=${version} -Darguments="-Dgpg.passphrase=${gpg_pass}" 
+sed -i "s/\(<version>\)[0-9]*\.[0-9]*\.[0-9]*\(<\/version>\)/\1${version}\2/" README.md
+git ci -a -m "bump version"
```
Comment (liry) [inline release.sh]: probably not everybody has this "ci" shortcut defined :)

Comment (jimirocks) [reply release.sh]: fixed

**Comment by liry** [comment]
@jimi Good job 👍

**Pull request merged**